widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -311,23 +312,27 @@ public void emitState(CallState callState) {
                             appBar.setTitle(resources.getString(R.string.call_audio_app_bar_title));
                         }
                     }
-                    operatorStatusView.isRippleAnimationShowing(callState.isCallNotOngoing());
+                    operatorStatusView.isRippleAnimationShowing(
+                            callState.isCallNotOngoing() ||
+                                    callState.isCallOngoig()
```

Comment:
typo

Reply:
is there btw any state where call is not ongoing and ongoing at the same time?

Reply:
It is || (OR), meaning when there is no media states pretty much..